mcc/odin/odin/mock_rds_client_test.go

```diff
@@ -190,15 +190,15 @@ func (m *mockRDSClient) CreateDBSnapshot(
 	if *instance.DBInstanceStatus != "available" {
 		err = fmt.Errorf(
 			"%s instance state is not available",
-			instanceID,
+			*instanceID,
```

Comment:
These two `*` where lacking (my mistake).
Since they were not there, these messages were messy, like `Snapshot (s!string0x04869384) already exists`.
Now, the `Errorf` function gets the string content, instead of the pointer.